### DISCUSSION TITLE
Hi

### DISCUSSION THREAD

**Discussion by M1t3Sh** [DhiWise/dhiwise-nodejs#38] [General]
Thanks for the node generator.

Could you please explain why the generator is having admin and device two directory to cater the api? That would help to understand the process

**Comment by saloni137** [comment]
@M1t3Sh Are you talking about generated code? 
FYI, you can check about DhiWise nodejs generator here https://docs.dhiwise.com/docs/node/intro

**Reply by M1t3Sh** [reply]
yes generated code, My Q is there are two folders "device" and "admin" and both contains apis, but what is the purpose of having folder name device? Thanks

**Reply by saloni137** [reply]
We are providing Platform wise APIs. so Basically, you can get APIs for Admin, client and mobile platforms separately for different types of user access. You can remove these platforms while configuring your app from GUI. One platform will always remain.
